hack/push-container-manifest.sh

```diff
@@ -34,6 +34,9 @@ function podman_push_manifest() {
     ${KUBEVIRT_CRI} manifest create ${DOCKER_PREFIX}/${image}:${DOCKER_TAG}
     for ARCH in ${BUILD_ARCH//,/ }; do
         FORMATED_ARCH=$(format_archname ${ARCH})
+        if [ ! -f ${DIGESTS_DIR}/${FORMATED_ARCH}/bazel-bin/push-$image.digest ]; then
```

Comment:
I would sleep better if we make this explicit.

Reply:
I'm afraid this would make this script a lot bigger and forces us to either redefine the images from `bazel-push-images.sh` here or move them to a new script we source.
This change is much more simple instead.
Though to be fair it comes with the downside that if the cache already has a digest for an image you no longer want you need to manually delete it...

Reply:
Can you share how does this fail today?

Reply:
Ah, I see. I was under the expression we list the images per arch but now I see we list all images and then have assumption to push them for all arch...